src/components/shared/NewEventForm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+import Fab from '@material-ui/core/Fab';
+import AddIcon from '@material-ui/icons/Add';
+import Tooltip from '@material-ui/core/Tooltip';
+
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
+    margin: '30%',
+    padding: '30%'
+  }
+});
+
+export default function NewEventForm() {
```

Comment:
One alternative is what's called named import (which what we have been doing for other applications as well). It looks like this `export function NewEventForm(){...}` and it can be imported like `import { NewEventForm } from '.....';`. [here](https://medium.com/@timoxley/named-exports-as-the-default-export-api-670b1b554f65) is a good article which breakdowns the differences between named and default exports.